readme.md

```diff
@@ -315,7 +315,7 @@ Thanks for contributing! 🦋🙌
 ### Commits
 
 - [](# "patch-diff-links") [Adds links to `.patch` and `.diff` files in commits.](https://cloud.githubusercontent.com/assets/737065/13605562/22faa79e-e516-11e5-80db-2da6aa7965ac.png)
-- [](# "raw-file-link") [Adds a link to view the raw version of files in PRs and commits.](https://user-images.githubusercontent.com/1402241/56484988-b99f2500-6504-11e9-9748-c944e1070cc8.png)
+- [](# "raw-and-blame-file-links") [Adds links to view the raw version and the blame of files in PRs and commits.](https://user-images.githubusercontent.com/46634000/144711364-26c75963-448c-4a5c-8f32-96d2c74ff7e4.png)
```

Comment:
pr-file-links?

Reply:
I'm following the logic of `scheduled-and-manual-workflow-indicators` :wink:

Reply:
This takes too many bytes in my Safari memory 😣

That works, but I'm concerned that we also need to add the history link and https://github.com/refined-github/refined-github/issues/3289, so that'd start to be a lot. I'm not sure if these appear just on PRs though.

Reply:
It appears on isCompare but I don't see an issue with both being there.